Dockerfile

```diff
@@ -6,7 +6,7 @@ FROM phusion/baseimage:0.9.16
 MAINTAINER Stuart Munro <stuart.munro@digital.justice.gov.uk>
 
 # Runtime User
-RUN useradd -m -d /home/app app
+RUN useradd --uid 1000 --user-group -m -d /home/app app
```

Comment:
I'm a bit confused here. The `app` user would originally have a non-root user ID and on ubuntu it defaults to `1000`, so the uid change seems redundant.

If I execute the `id` command on the current image, I get back
```
$ id
uid=1000(app) gid=1000(app) groups=1000(app)
```
Am I missing something?

Reply:
Right, given the answer to the above question it's good practice to be explicit about the uid being created than using implicit defaults. 👍

Reply:
Ah ok. This is probably redundant, then, but it looked like the right solution given the error we were experiencing. I guess it might only be the above `USER <id>` vs `USER <name>` bit of it that did it in the end.

Reply:
I like that this command is explicit given that we must provide a numeric ID.

Reply:
Yep, agreed